docs/concepts/policy/pod-security-policy.md

```diff
@@ -125,141 +399,153 @@ to the volume sources that are defined when creating a volume:
 The recommended minimum set of allowed volumes for new PSPs are
 configMap, downwardAPI, emptyDir, persistentVolumeClaim, secret, and projected.
 
-### Host Network
- - *HostPorts*, default `empty`. List of `HostPortRange`, defined by `min`(inclusive) and `max`(inclusive), which define the allowed host ports.
-
-### AllowPrivilegeEscalation
-
-Gates whether or not a user is allowed to set the security context of a container
-to `allowPrivilegeEscalation=true`. This field defaults to `false`.
-
-### DefaultAllowPrivilegeEscalation
-
-Sets the default for the security context `AllowPrivilegeEscalation` of a container.
-This bool directly controls whether the `no_new_privs` flag gets set on the
-container process. It defaults to `nil`. The default behavior of `nil`
-allows privilege escalation so as to not break setuid binaries. Setting it to `false`
-ensures that no child process of a container can gain more privileges than
-its parent.
+**FSGroup** - Controls the supplemental group applied to some volumes.
 
-### AllowedHostPaths
+- *MustRunAs* - Requires at least one range to be specified. Uses the
+minimum value of the first range as the default. Validates against the
+first ID in the first range.
+- *RunAsAny* - No default provided. Allows any `fsGroup` ID to be specified.
 
-This specifies a whitelist of host paths that are allowed to be used by Pods.
-An empty list means there is no restriction on host paths used.
-Each item in the list must specify a string value named `pathPrefix` that
-defines a host path to match. The value cannot be "`*`" though.
-An example is shown below:
+**AllowedHostPaths** - This specifies a whitelist of host paths that are allowed
+to be used by HostPath volumes. An empty list means there is no restriction on
```

Comment:
Consider replacing `HostPath` by `hostPath` in this paragraph.

Reply:
done.